image_client/picturae_importer.py

```diff
@@ -24,10 +24,10 @@ class PicturaeImporter(Importer):
            along with attached images
     """
 
-    def __init__(self, paths, date_string=None, istest=False):
+    def __init__(self, paths, date_string=None):
         super().__init__(picturae_config, "Botany")
 
-        self.setting_init_variables(date_string=date_string, paths=paths)
+        self.init_all_vars(date_string=date_string, paths=paths)
```

Comment:
made init_all_vars function name same as analagous function in CsvPicturaeCreate